app/components/dependency-table.hbs

```diff
@@ -18,8 +18,8 @@
           {{#each this.devDependencies as |dependency index|}}
             {{#if (or this.showingAllDependencies (lt index this.limit))}}
               <tr>
-                <td class="test-dependency-name">
-                  <LinkTo @route="addons.show" @model={{dependency}}>{{dependency}}</LinkTo>
+                <td class={{concat "test-dev-dependency-" dependency.packageName}}>
```

Comment:
No need for `concat` here:
`<td class="test-dev-dependency-{{dependency.packageName}}">`

Reply:
Fixed in both spots.